package.json

```diff
@@ -39,6 +40,7 @@
   "author": "rt2zz <zack@root-two.com>",
   "license": "MIT",
   "dependencies": {
+    "@types/redux": "^3.6.0",
```

Comment:
It will not, there are no js file under `@types/xxx`.
However, it is fine to move it to devDeps, for end user, `@types/redux` will be added with `redux`.